.travis.yml

```diff
@@ -4,8 +4,23 @@ go:
   - 1.8
 
 dist: trusty
-sudo: false
+sudo: required
```

Comment:
`docker build` needs the docker daemon to be running. Unfortunately, it looks like we can not do without `sudo: required`.